Sources/MetricKit/MetricsMonitor.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  MetricsMonitor.swift
+//  
+//
+//  Created by Antoine van der Lee on 31/08/2021.
+//
+
+import Foundation
+import MetricKit
+
+/// Monitors payloads delivered by MetricKit and logs valueable information, including crashes.
+final class MetricsMonitor: NSObject {
+
+    func startMonitoring() {
+        if #available(iOS 14, *) {
+            MXMetricManager.shared.add(self)
+        }
+
+        NSSetUncaughtExceptionHandler { exception in
+            MetricsMonitor.logExceptionUsingCallStackSymbols(exception, description: "Uncaught Exception")
```

Comment:
Do I understand it correctly we now log crashes the old way and new way through metric kit? Because metric kit is only delivering reports once every 24 hours.

Reply:
Yes! MetricKit only delivers once per 24 hours, so you're not sure whether a crash is reported. Though, MetricKit reports much more than this exception handler. For example, `EXC_BAD_ACCESS` crashes don't get catched with this handler as there's no exception thrown.

Therefore, a combination of both is most useful